autodoc/admin-api/data/admin-api.lua

```diff
@@ -810,8 +810,8 @@ return {
         },
         protocols = {
           description = [[
-            A list of the protocols this Route should allow. When set to `["https"]`,
-            HTTP requests are answered with a request to upgrade to HTTPS.
+            An array of the protocols this Route should allow. To allow only one protocol, send a string value. See the [Route Object](#route-object) section for a list of accepted protocols. When set to only `"https"`,
```

Comment:
>  To allow only one protocol, send a string value.

The "string" to array conversion is only supported with `application/x-www-form-urlencoded` and `multipart/form-data`, not with `application/json`.

Reply:
resolved with commit `7dc2213`